platform/android/export/export.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "export.h"
+#include "gradle_export_util.h"
```

Comment:
This should be in the block below. Only the file's corresponding header is listed separately (`export.h` for `export.cpp`): https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes